client/packages/system/src/Name/DetailModal/DetailModal.tsx

```diff
@@ -68,16 +68,30 @@ export const DetailModal: FC<DetailModalProps> = ({ nameId }) => {
             <DetailInputWithLabelRow
               label={t('label.website')}
               inputProps={{ value: data?.website, disabled: isDisabled }}
+              inputAlignment="start"
               DisabledInput={
-                <>
-                  <MuiLink
-                    href={data.website ?? undefined}
-                    target="_blank"
-                    rel="noopener"
-                  >
-                    {data.website}
-                  </MuiLink>
-                </>
+                <MuiLink
+                  href={data.website ?? undefined}
+                  target="_blank"
+                  rel="noopener"
+                  sx={{
+                    // truncate long links
+                    overflow: 'hidden',
+                    textWrap: 'nowrap',
+                    textOverflow: 'ellipsis',
```

Comment:
I think it would be nice to wrap the website and show full URL? Think it should be fine if users don't start entering some weird super long URL.... @mark-prins what do you think? 👀 
![Screenshot 2024-05-08 at 14 05 43](https://github.com/msupply-foundation/open-msupply/assets/61820074/11af8e9d-c614-4658-a909-9734b3541f87)